net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+MY_PN="dnsviz"
+
+DESCRIPTION="Tool suite for analysis and visualization of DNS and DNSSEC"
+HOMEPAGE="http://dnsviz.net/"
+SRC_URI="https://github.com/dnsviz/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="bind"
```

Comment:
This "bind" IUSE doesn't seem to have any other function than be in RDEPEND. Please drop it, Gentoo doesn't use USE flags for runtime dependencies since if you change it from "bind" to "-bind", it will trigger a rebuild but nothing with installed files will change. Use `optfeature` instead and move the USE flag description from metadata.xml there :)

Reply:
I had some confusion about that already.Thank you for clear answer.My logic always works "integrate all options to ebuild for user with flags" but it is not possible always. 

Package has extra options if bind & gost engine exist.
I tried to integrate gost-engine but it depends openssl 1.1.1 so not possible for now.
Same for BIND.It is not DEPEND, RDEPEND, BDEPEND even does not need to be running as a server.

I will add optfeature for both of them.